src/components/Badge/Badge.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { resizeImage } from '@quartz/js-utils';
+import Image from '../Image/Image';
+import styles from './Badge.scss';
+
+const sizeToPx = {
+	'extra-large': 82,
+	large: 64,
+	medium: 50,
+	small: 40,
+};
+
+function Badge ( {
+	alt,
+	imageUrl,
+	size,
+} ) {
+	// Convert size prop to image dimension. All badges are (rounded) squares.
+	const px = sizeToPx[ size ];
+
+	// Create an srcSet for 1x, 2x and 3x resolution.
+	const sizes = `${px}w`;
+	const srcSet = [ 1, 2, 3 ].map( resolution => `${resizeImage( imageUrl, px * resolution, px * resolution, true )} ${resolution}x` ).join( ', ' );
+	const src = resizeImage( imageUrl, px, px, true );
+
+	return (
+		<figure className={`${styles.container} ${styles[ size ]}`}>
```

Comment:
I don't know that figure is the right tag for this. This component seems more geared towards decoration that content. WDYT?

Reply:
Yeah I wasn't sure either. I think just a div here is better